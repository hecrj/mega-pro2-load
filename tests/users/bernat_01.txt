7

10	
20	
50	
35
40	
90	
22	

9

2 3 8 0 0 9 0 0 5 1 0 7 6 0 0 0 4 0 0 

12	
3 1 3 5	
25	
2 5 6 	
20
4 2 5 6 7
5
2 3 4
7
2 2 3
8 
2 4 5
50
2 1 2
3
2 6 7
3	
2 6 7	

-3 1 	

-5 2 5 	

-2 2 	

-1 5 5	

-1 4 9	

-1 4 10 

-1 4 11 

-3 12	

-2 13	

-3 14	

-5 5 9 	

-5 10 10 

-1 6 15	

-2 16	

-4 3 0 1 5	

-4 7 1 6 0	

-1 6 17		

-2 19

-1 1 20

-1 7 22

-6

