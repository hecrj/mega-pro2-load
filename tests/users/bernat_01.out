Servidores ocupados
0
Pelicula mas solicitada
0 0
Peticiones pendientes
0
Peticion procesada y servidores
0 1
2 3 
Peticion procesada y servidores
1 5
6 
Peticion procesada y servidores
2 7
4 
Peticion procesada y servidores
3 0
Servidores ocupados
4 2 5
6 1 2
Peticiones pendientes
1 4 9 14
2 4 10 17
Servidores ocupados
4 2 3
Pelicula mas solicitada
4 1
Pelicula mas solicitada
4 1
Peticion procesada y servidores
4 2
2 3 8 
Peticiones pendientes
2 4 10 17
4 6 15 17
Peticion procesada y servidores
5 2
2 7 
Peticiones pendientes
0
Peticion procesada y servidores
6 1
1 
Peticion procesada y servidores
7 1
3 8 
